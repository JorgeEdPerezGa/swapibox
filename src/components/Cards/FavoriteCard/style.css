p {
  margin-left: 20px; }

.card {
  width: 300px;
  background: white;
  margin: 30px;
  text-align: left; }

.card-title-favorite {
  background: #430f6f;
  padding-right: 50px; }
