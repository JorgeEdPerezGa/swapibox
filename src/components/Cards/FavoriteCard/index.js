import React from 'react';
import FavoriteButton from '../../FavoriteButton';
import './style.css';

const FavoriteCard = ({ card, name, species, homeworld, homeworldPopulation, climate, population, residents, model, vehicleClass, passengers, favorite, handleFavorites }) => {
  const favortieClass = favorite ? 'favorited' : '';
  return (
    <article className = 'card'>
      <FavoriteButton
        favortieClass = { favortieClass }
        handleFavorites = { handleFavorites }
        card = { card }/>
      <h1 className = 'card-title card-title-favorite'>{ name }</h1>
      <p>{ species }</p>
      <p>{ homeworld }</p>
      <p>{ homeworldPopulation }</p>
      <p>{ climate }</p>
      <p>{ population }</p>
      <p>{ residents }</p>
      <p>{ model }</p>
      <p>{ vehicleClass }</p>
      <p>{ passengers }</p>
    </article>
  );
};

export default FavoriteCard;
