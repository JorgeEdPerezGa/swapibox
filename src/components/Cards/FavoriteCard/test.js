import React from 'react';
import FavoriteCard from './index';
import setupTest from '../../../setupTests';
import { shallow } from 'enzyme';

describe('<FavoriteCard/>', () => {
  const wrapper = <FavoriteCard />;

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
