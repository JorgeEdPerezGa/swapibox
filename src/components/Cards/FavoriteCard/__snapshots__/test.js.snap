// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoriteCard/> should match snapshot 1`] = `<FavoriteCard />`;
